apps/frontend/src/app/5_pages/MarketMakingPage/hooks/useGetExpectedTokenAmount.ts

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useMemo, useState, useCallback } from 'react';
+
+import { Contract } from 'ethers';
+
+import { SupportedTokens, getTokenContract } from '@sovryn/contracts';
+import { getProvider } from '@sovryn/ethers-provider';
+import { Decimal } from '@sovryn/utils';
+
+import { defaultChainId } from '../../../../config/chains';
+
+import { asyncCall } from '../../../../store/rxjs/provider-cache';
+import { AmmLiquidityPool } from '../utils/AmmLiquidityPool';
+
+export const useGetExpectedTokenAmount = (
+  pool: AmmLiquidityPool,
+  amount: Decimal,
+) => {
```

Comment:
```suggestion
): Decimal => {
```
Type it so it doesn't show as `any`.